### PR TITLE
executor/brie: use the default value from flags

### LINKED ISSUE #48000: br: disable the stats information backup by default for sql command
## Enhancement
Stats information backup needs to call `dumpStatsToJson` function which costs a lot of memory especially when there are a lot of database objects.

For br clp, flag `ignore-stats` is turned on by default, while it's not for backup sql command. So, backup could bring OOM issues.

We need to sync the behavior to as the clp, and also expose the parameter to sql command.

Via this issue, we also needs to sync up the default flags including `use-checkpoint`.

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve?

Issue Number: close #48000

Problem Summary:
The current default config parameter is different from the BR CLI's. For example, `SkipStatistics` is always `false` (in BR it is defaultly `true`), `UseCheckpoint` is always `false` too (in BR it is defaultly `true`).

### What is changed and how it works?
This PR exports the default config via the command line parameter of BR. And we will use it at the BRIE via SQL.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Now, BRIE via SQL will use the same default config with BR CLI.
```

### CODE REVIEW

**Comment by BornChanger** [comment]
please associate it with issue #48000 @YuJuncen

**Comment by BornChanger** [comment]
please also provide those 2 parameters to BR sql command option parser. @YuJuncen

File: pkg/executor/brie.go [hunk 1]
```diff
@@ -392,7 +388,9 @@ func (b *executorBuilder) buildBRIE(s *ast.BRIEStmt, schema *expression.Schema)
 		}
 
 	case ast.BRIEKindRestore:
-		e.restoreCfg = &task.RestoreConfig{Config: cfg}
+		rcfg := task.DefaultRestoreConfig()
+		rcfg.Config = cfg
```
Comment (Leavrth) [inline pkg/executor/brie.go:392]: It seems that `rcfg.Config` has been parsed in the `task.DefaultRestoreConfig()`.

**Review by Leavrth — APPROVED** [review]

File: pkg/executor/brie.go (hunk 1 above)
Comment (YuJuncen) [reply pkg/executor/brie.go:392]: We will override the default config with the new config. Will this make things wrong?

**Review by BornChanger — APPROVED** [review]

**Comment by YuJuncen** [comment]
√

**Pull request merged**